arch/Mips/MipsModule.c

```diff
@@ -9,6 +9,15 @@
 #include "MipsInstPrinter.h"
 #include "MipsMapping.h"
 
+// Returns mode value with implied bits set
+static inline cs_mode updated_mode(cs_mode mode) {
```

Comment:
can you put this bracket on the new line, to be consistent with the current coding style?